src/authextension.ts

```diff
@@ -7,11 +7,11 @@ export module authextension {
   export let setContextData: any;
 
   setContextData = (context_f8_access_routes, context_f8_3scale_user_key) => {
-    Apiendpoint.STACK_API_URL = context_f8_access_routes.prod + '/api/v1/';
+    Apiendpoint.STACK_API_URL = context_f8_access_routes.prod + '/api/v2/';
     Apiendpoint.STACK_API_USER_KEY = context_f8_3scale_user_key;
     Apiendpoint.OSIO_ROUTE_URL = context_f8_access_routes.prod;
     process.env['RECOMMENDER_API_URL'] =
-      context_f8_access_routes.prod + '/api/v1';
+      context_f8_access_routes.prod + '/api/v2';
     process.env['THREE_SCALE_USER_TOKEN'] = context_f8_3scale_user_key;
```

Comment:
The User token should be updated with respect to stage for v2 api.

Reply:
What is that user token @dvandra ?

Reply:
I mean to say the `f8_3scale_user_key` for stage.

Reply:
According to this : https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/pull/375#discussion_r443338028

It looks good to me.